pkg/api/http/err.go

```diff
@@ -53,3 +51,51 @@ func NewErrStatusCodeFromResponse(resp *http.Response) error {
 
 	return errsc
 }
+
+func NewErrStatusCode(statusCode int, msg string, errorCode ...string) ErrStatusCode {
```

Comment:
The intention behind the variadic args was for making it easier to format the message!

```suggestion
func NewErrStatusCode(statusCode int, msg string, args ...string) ErrStatusCode {
```